### ISSUE TITLE
Same TARGETID,RA,DEC assigned to different simulated targets in MAPLE sim

### ISSUE THREAD

**Issue by cballand** [desihub/desisim#141]
Different targets (with different type and redshift) have been assigned the same TARGETID and (RA,DEC) in different exps of the MAPLE simulation. 

As an example, TARGETID=5755267226444861799 corresponds to an ELG at z=1.55605 in exp 7 of night 20160607 and to a STAR at z=0.0005 in exp 13 of the same night:
```
from astropy.io import fits
simspec7 = fits.getdata('/scratch1/scratchdirs/sjbailey/desi/spectro/sim/maple/20160607/simspec-00000007.fits','METADATA')
simspec13 = fits.getdata('/scratch1/scratchdirs/sjbailey/desi/spectro/sim/maple/20160607/simspec-00000013.fits','METADATA')
fibermap7 = fits.getdata('/scratch1/scratchdirs/sjbailey/desi/spectro/sim/maple/20160607/fibermap-00000007.fits',1)
fibermap13 = fits.getdata('/scratch1/scratchdirs/sjbailey/desi/spectro/sim/maple/20160607/fibermap-00000013.fits',1)
j, = np.where(fibermap7['TARGETID']==5755267226444861799)
k, = np.where(fibermap13['TARGETID']==5755267226444861799)
print j,k,fibermap7['TARGETID'][j],fibermap13['TARGETID'][k],simspec7['REDSHIFT'][j],simspec13['REDSHIFT'][k],simspec7['OBJTYPE'][j],simspec13['OBJTYPE'][k]

[978] [980] [5755267226444861799] [5755267226444861799] [ 1.55605257] [ 0.00049941] ['ELG'] ['MWS_STAR']
```
The two targets with TARGETID=5755267226444861799 are plotted below from the bricks: brick-{b,r,z}-3125p100.fits. The lower spectrum is an ELG at z=1.55605 (small [OII] emission is visible around 9526 A when zooming).

![bug_maple](https://cloud.githubusercontent.com/assets/15869020/16658773/dd6afb1e-4467-11e6-9104-cc77ff929a36.png)

I've checked that the hemlock simulation does not have this bug.

**Comment by julienguy** [comment]
This bug prevents us from using the output of redmonster from the maple simulation run because redmonster uses all exposures of a given targetid (a mix of spectra with this bug) in a brick to fit the redshifts.

It would be useful to fix this soon and rerun the pixel level simulations now that the pipeline runs smoothly.
This would be useful to check efficiencies, failures, and design a lookup table for quickcat (Christophe and Bob are working on it) on more stats. than zDC1 with all the pipeline artifacts (cosmic rays, sky & calibration errors).

**Comment by sbailey** [comment]
Argh, I missed seeing this go by.  This was fixed with PR #133 (unless there is another bug with the same symptom), but I had forgotten that madrone/maple were generated before that fix was in.  I will generate a new simulation and check for consistency of duplicates.  I'll post the new simulation location to this ticket before closing it.

**Comment by sbailey** [comment]
Update:
* this affects maple (9 exposures x 10 spectrographs) but not madrone (3 exposures x 2 spectrographs)
* it was not fixed by PR #133; this is a new bug
* it is not the explanation of the low redmonster performance on madrone QSOs reported at the 2016-07-12 DESI data telecon

Investigating...
